Shape_detection/doc/Shape_detection/Shape_detection.txt

```diff
@@ -565,16 +642,11 @@ which are subgroups of axis-symmetric and orthogonal clusters) as described by V
 
 \section Shape_detection_DeprecatedComponents Deprecated Components
 
-The new version (see all examples above) of the class `CGAL::Shape_detection::Region_growing`
-is not compatible with the old API. For the old API, see an example below and use
-`CGAL::Shape_detection::deprecated::Region_growing_depr` instead.
-
-\cgalExample{Shape_detection/shape_detection_basic_deprecated.cpp}
-
-The old API is still supported, but will be removed in the next releases.
-In particular, the classes `CGAL::Shape_detection::deprecated::Region_growing_depr` and
-`CGAL::Shape_detection::deprecated::Shape_detection_traits` will be removed.
-Please update your code.
+This version of Region Growing deprecates the old way to specify parameters. Now, all parameters
+are provided as \ref bgl_namedparameters "Named Parameters". That means the old constructors are deprecated.
+Second, the method `update` of the \ref Shape_detection_RegionGrowingFramework_conditions "RegionType" concept
+now returns a Boolean flag. When it is `true`, the new region is further propagated, otherwise, it is rejected.
+The reason behind this change is to provide users with control for early rejection of non-valid regions.
```

Comment:
Maybe add a quick code example to show how to switch to the new API?

I think there's a way to display a small portion code without actually creating a new example (check Doxygen), something very simple like:

```c++
// Old API
Region_growing region_growing (points, points_map, blabla);

// New API
Region_growing region_growing (points, CGAL::parameters::point_map(point_map).blabla(blabla));
```